shared/ui/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,49 @@
+import { ComponentPropsWithoutRef } from 'react';
+
+import { Typography } from '@/shared/ui';
+import * as Radio from '@radix-ui/react-radio-group';
+import { clsx } from 'clsx';
+
+import style from './radio-group.module.scss';
+
+export type OptionsRadioGroup = {
+  disabled?: boolean;
+  id?: string;
+  label?: string;
+  value: string;
+}[];
+
+export type RadioGroupProps = {
+  className?: string;
+  options: OptionsRadioGroup;
+} & Omit<ComponentPropsWithoutRef<typeof Radio.Root>, 'children'>;
+
+export const RadioGroup = ({ className, options, ...rest }: RadioGroupProps) => {
+  return (
+    <Radio.Root
+      className={clsx(style.radioGroupRoot, className)}
+      defaultValue={options[0].value}
```

Comment:
This assigment doen't work. You override this statement on the next line by passing `{...rest}`. So this means
```typescript`
<RadioGroup 
  defaultValue={'anything'} // 'anything' will override options[0].value
  options={[]}
/>